test/test_grade.py

```diff
@@ -63,6 +62,8 @@ def generate_autograder_zip(pdfs=False):
             config = TEST_FILES_PATH + "otter_config.json" if pdfs else None,
             no_environment = True,
         )
+        with zipfile.ZipFile(TEST_FILES_PATH + "autograder.zip", "a") as zip_ref:
+            utils.zip_folder(zip_ref, os.path.dirname(os.path.dirname(os.path.abspath(__file__))), exclude=[".git", "logo", "test"])
```

Comment:
I'm not sure this block is working as intended. I cloned your fork of the repo and ran a test to see if the local version of Otter is being used by changing the version number in `otter/version.py` and the change was not reflected.

e.g. change `otter.version.__version__` to `"4000.0.0"`, run the test `python3 -m test grade timeout`, and note that the autograder stdout shows Otter v3.1.4, not the updated version number. I think something is going wrong with how the development version of Otter is being installed.

Reply:
I will check that asap.

Reply:
Sorry, but I'm not able to reproduce the error. I tried it on different systems and the provided test (changing the version number). It works fine everywhere. 

![grafik](https://user-images.githubusercontent.com/32478819/135621300-5921a58f-a4ac-44b5-8bcf-2d4c21eedf83.png)

Also I don't really know where the syntax error should be ... Both lines should add the curent dev version to the zip, this should be extracted again, added using the `ADD .  /autograder/source` and installed...

Reply:
Hi, 

I've tested the changes on some other systems and - sorry - but I can not reproduce the bug ... Maybe you can provide me some more details so I can debug it?